python/lsst/meas/extensions/multiprofit/rebuild_coadd_multiband.py

```diff
@@ -217,7 +217,7 @@ def from_butler(
                 catalog=catalog,
                 band=data_id["band"],
                 imageForRedistribution=exposure,
-                removeScarletData=True,
+                removeScarletData=False,
```

Comment:
I think you mentioned that to accomplish the original goal of this line, you have to call... `models_scarlet.clear()` after the loop over bands? Is that right?

Reply:
You don't have to call that, and I think it would be something like `models_scarlet.blends.clear()`. I didn't bother because `models_scarlet` doesn't exist outside the scope of this method (I might not have realized that when we talked about it previously) so it will be garbage collected once this function is completed anyway.